docs/tutorial3.md

```diff
@@ -140,7 +145,7 @@ Full Source Code
 
 ```C++
 #include <sc2api/sc2_api.h>
-
+#include <sc2api/sc2_unit_filters.h>
```

Comment:
According to the Google C++ style guide you should group sc2api imports together. Thus it should be:
```
```C++
#include <sc2api/sc2_api.h>
#include <sc2api/sc2_unit_filters.h>

#include <iostream>
```